src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../react-app-env';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './TodoInfo.scss';
+
+interface Props {
+  todo: Todo,
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={`todoInfo ${todo.completed ? 'todoInfo--completed' : ''}`}
```

Comment:
use the following tool instead, ternary with empty string is not a proper solution
https://www.npmjs.com/package/classnames

Reply:
I'd tried to made it with classnames, but recived errors during instalation😥